src/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -238,7 +244,7 @@ def _get_native_state(self):
         # We don't know how to serialize a Python class, hence we need to
         # reduce the detector to the proxy object before saving it.
         if USE_CPP:
-            pass
+            state['detector'] = {'detector_model':state['detector'].__proxy__}
```

Comment:
Why do we need this?

Reply:
https://github.com/apple/turicreate/blob/e91c16f012c7bb96106209b655502208b48a9c48/src/python/turicreate/toolkits/_model.py#L122
Here we need to know the OD model is saved in version 5.8 (or older) or in the 6.0 version.

Reply:
Ah, got it. I keep getting confused between the OSOD wrapper class and the inner OD class